Makefile.generated_files

```diff
@@ -12,8 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
That's super annoying.  In my env it seems to get passed through because of .EXPORT_ALL_VARIABLES.

Have I ever mentioned how I feel about MacOS ?  Bringing UNIX back to the 70s.

I have re-pushed with explicit SHELL setting in the secondary Makefile.